### ISSUE TITLE
operator: example uses broken image

### ISSUE THREAD

**Issue by pohly** [intel/pmem-csi#576]
```
$ test/start-operator.sh 
Deploying '/nvme/gopath/src/github.com/intel/pmem-csi/operator/deploy/operator.yaml'...
serviceaccount/pmem-csi-operator unchanged
role.rbac.authorization.k8s.io/pmem-csi-operator configured
clusterrole.rbac.authorization.k8s.io/pmem-csi-operator unchanged
rolebinding.rbac.authorization.k8s.io/pmem-csi-operator configured
clusterrolebinding.rbac.authorization.k8s.io/pmem-csi-operator unchanged
deployment.apps/pmem-csi-operator created
PMEM-CSI operator is running. To try out deploying the pmem-csi driver:
    cat operator/examples/pmem-csi.intel.com_v1alpha1_deployment_cr.yaml | /nvme/gopath/src/github.com/intel/pmem-csi/_work/pmem-govm/ssh.0 kubectl create -f -
cleaning up temp folder: /tmp/tmp.QFkHuVNfqB
$ cat operator/examples/pmem-csi.intel.com_v1alpha1_deployment_cr.yaml | /nvme/gopath/src/github.com/intel/pmem-csi/_work/pmem-govm/ssh.0 kubectl create -f -
```

This should bring up the driver, but that currently fails because operator/examples/pmem-csi.intel.com_v1alpha1_deployment_cr.yaml hard-codes an image that cannot be pulled (uses `172.17.0.1:5000`, which isn't correct for me).

It would be better to not use an image and then let the operator figure it out.

**Comment by avalluri** [comment]
Fixed in #584

**Comment by pohly** [comment]
Fixed.
